elasticlib/Elastic.java

```diff
@@ -0,0 +1,53 @@
+package frc.robot;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.PubSubOption;
+import edu.wpi.first.networktables.StringArrayPublisher;
+import edu.wpi.first.networktables.StringArrayTopic;
+import edu.wpi.first.networktables.StringPublisher;
+import edu.wpi.first.networktables.StringTopic;
+
+public final class Elastic {
+    private static final StringTopic topic = NetworkTableInstance.getDefault().getStringTopic("elastic/robotalerts");
+    private static final StringPublisher publisher = topic.publish(PubSubOption.sendAll(true));
+    private static final ObjectMapper objectMapper = new ObjectMapper();
+
+    public static void sendAlert(RobotAlert alert) {
+        try {
+            publisher.set(objectMapper.writeValueAsString(alert));
```

Comment:
I haven't looked much into the Jackson json encoding much yet, but I believe there's some annotations that need to be added to the notification class for it to be encoded properly